pyrsa/__init__.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+""" Top level package: Only imports and organisation
+"""
+import pyrsa.model as model
```

Comment:
I don't think this is necessary?

Reply:
pylint complaint about it when it was missing on my computer. Would not think this is necessary either

Reply:
It complained about a missing module? Did you try a `python setup.py develop`?

Reply:
I was talking about the comment. Are you talking about the import?

Reply:
yes

Reply:
tests still run if your remove it

Reply:
yes you are right -> removed